src/test/resources/test_data.txt

```diff
@@ -285,20 +284,20 @@ Polyrhachis orsyllus nat musculus Forel 1901|{"scientificName":{"id":"3392132e-3
-Mycosphaerella eryngii (Fr.ex Duby) ex Oudem. 1897|{"scientificName":{"id":"a2c92db9-2bac-59b0-8e96-6a885c30eb9b", "parsed":true, "parser_version":"test_version", "parser_run":1,"verbatim":"Mycosphaerella eryngii (Fr.ex Duby) ex Oudem. 1897","normalized":"Mycosphaerella eryngii (Fr. ex Duby) ex Oudem. 1897","canonical":"Mycosphaerella eryngii", "hybrid":false, "virus": false, "details":[{"genus":{"string":"Mycosphaerella"},"species":{"string":"eryngii","authorship":"(Fr.ex Duby)","basionymAuthorTeam":{"author":["Fr.ex Duby"]}},"infraspecies":[{"string":"ex","rank":"n/a","authorship":"Oudem. 1897","basionymAuthorTeam":{"author":["Oudem."],"year":"1897"}}]}],"positions":{"0":["genus",14],"15":["species",22],"24":["author_word",29],"30":["author_word",34],"36":["infraspecies",38],"39":["author_word",45],"46":["year",50]}}}
-Salmonella werahensis (Castellani) Hauduroy and Ehringer in Hauduroy 1937|{"scientificName":{"id":"bb6e2a9f-6813-5b00-9a3f-e12a085e515e", "parsed":true, "parser_version":"test_version", "parser_run":1,"verbatim":"Salmonella werahensis (Castellani) Hauduroy and Ehringer in Hauduroy 1937","normalized":"Salmonella werahensis (Castellani) Hauduroy & Ehringer ex Hauduroy 1937","canonical":"Salmonella werahensis", "hybrid":false, "virus": false, "details":[{"genus":{"string":"Salmonella"},"species":{"string":"werahensis","authorship":"(Castellani) Hauduroy and Ehringer in Hauduroy 1937","combinationAuthorTeam":{"author":["Hauduroy","Ehringer"],"exAuthorTeam":{"author":["Hauduroy"],"year":"1937"}},"basionymAuthorTeam":{"author":["Castellani"]}}}],"positions":{"0":["genus",10],"11":["species",21],"23":["author_word",33],"35":["author_word",43],"48":["author_word",56],"60":["author_word",68],"69":["year",73]}}}
+Glomopsis lonicerae Peck ex C.J. Gould 1945|{"scientificName":{"id":"422687ca-7f4b-5720-8d99-88695f765530","parsed":true,"parser_version":"test_version","verbatim":"Glomopsis lonicerae Peck ex C.J. Gould 1945","normalized":"Glomopsis lonicerae Peck ex C. J. Gould 1945","canonical":"Glomopsis lonicerae","hybrid":false,"virus":false,"details":[{"genus":{"string":"Glomopsis"},"species":{"string":"lonicerae","authorship":"Peck ex C. J. Gould 1945","basionymAuthorTeam":{"author":["Peck"],"year":"1945","exAuthorTeam":{"author":["C. J. Gould"]}}}}]}}
```

Comment:
If `"parser_run":1` is not required then it should be removed in all test cases in a separate PR.